packages/odyssey-react/src/components/Select/Select.module.scss

```diff
@@ -494,6 +494,8 @@
   background-color: transparent;
   background-repeat: no-repeat;
   background-position: center;
+  font-family: inherit;
+  font-size: 100%;
```

Comment:
any reason font-family styles need to be redefined to inherit here and below? Not sure I follow why Select would need to?

Reply:
Yes. Choices renders a vanilla input at runtime that has no style for the multiselect variant. In order to get the component variant to look correct, we need to add this in. This was previously part of our CSS reset.

Reply:
got it - looking forward to rebuilding select soon and great job picking apart the global styles to find these hidden targets. This helps us tremendously in the future with maintenance because now everything is organized closer to where the styles are used